components/crypto/src/lib.rs

```diff
@@ -469,7 +469,7 @@ pub struct KeyPair {
 
 impl KeyPair {
     pub fn from_keys(public_key: PublicKey, secret_key: SecretKey) -> Self {
-        debug_assert!(
+        assert!(
```

Comment:
Why was this changed?

Reply:
Because this code breaks tests in release mode and @aleksuss asked me to fix this micro issue.